person.rb

```diff
@@ -1,21 +1,24 @@
+require './rentals'
+
 class Nameable
   def correct_name
     raise NotImplementedError
   end
 end
 
```

Comment:
- [ ] You should add a class Classroom and add all the associations

Reply:
@hiromataba A classroom is linked to the student only and not to the person. Could please show the connection